.golangci.yaml

```diff
@@ -2,38 +2,65 @@ run:
   deadline: 5m
 
 linters:
+  # Disable all linters.
+  disable-all: true
+  # Enable specific linters.
+  # https://golangci-lint.run/usage/linters/#enabled-by-default
   enable:
-  - revive
-  - gci
-  - depguard
-  - godot
-  - testifylint
-  - unconvert
+    - copyloopvar
+    - decorder
+    - gochecksumtype
+    - gofmt
+    - goheader
+    - goimports
+    - importas
+    - ineffassign
+    - misspell
+    - predeclared
+    - tagalign
+    - unconvert
+    - unused
+    - whitespace
```

Comment:
Curious: is there any reasoning for this specific list of linters? I usually just use the default set of linters unless I have a good reason, however it seems like there's a decent variety of enabled linters among open source projects

Reply:
I have updated the list of linters with the default set of linters enabled. Besides, I enabled some specific linters to format the code (like import order, tag align) and it might also help to improve the code quality (e.g. unused variable, unnecessary type conversions) without causing too much trouble for developers.

Reply:
LGTM. These can be refined over time if not behaving as intended